### PR TITLE
Update the development PSGI to cope with new location

### PR DESCRIPTION
**Pull request by ylavoie**
Make the development PSGI stay current.

### CODE REVIEW

File: utils/devel/ledgersmb-server-development.psgi [hunk 1]
```diff
@@ -16,9 +16,9 @@ package LedgerSMB::FCGI;
 no lib '.';
 
 use FindBin;
-use lib $FindBin::Bin . '/..'; # For our 'old code'-"require"s
-use lib $FindBin::Bin . '/../lib';
-use lib $FindBin::Bin . '/../old/lib';
+use lib $FindBin::Bin . '/../..'; # For our 'old code'-"require"s
+use lib $FindBin::Bin . '/../../lib';
+use lib $FindBin::Bin . '/../../old/lib';
```
Comment (ehuelsmann) [inline utils/devel/ledgersmb-server-development.psgi]: Why are these here at all? I mean, in the production version we depend on `-I` being passed to the calling process. I think that would be a good idea for the development infrastructure as well.

Comment (ylavoie) [reply utils/devel/ledgersmb-server-development.psgi]: Because we have the same in ledgersmb-server.psgi, only the paths were adjusted to cope with `/bin` and `/utils/devel`.

Comment (ylavoie) [reply utils/devel/ledgersmb-server-development.psgi]: I agree

**Comment by ehuelsmann** [comment]
Will you replace this PR with a removal-PR?
